docs/packages/connect/bump.md

```diff
@@ -0,0 +1,3 @@
+# Bump Connect in Suite
+
+TODO
```

Comment:
- change trezor-connect package version in `./packages/suite/package.json`
- run `yarn` (install new connect)
- run `yarn build:connect` - this step will build trezor connect files into `./packages/suite-data/files/connect`

Reply:
Added be6dcbe3c63d1b4a182ca68941cd518efb868589